JAVA_HOME is not set

Looking for a reliable way to start android emulators for CI testing and stumbled across this.  Unfortunately, it doesn't seem to be working in my environment.  I get the following error sequence (on a formerly working project), even if I explicitly define JAVA_HOME and java is in /usr/bin/java, on all the default paths on a Mac.

```
> Task :ui-android:installAndroidEmulatorSystemImage

ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.

Please set the JAVA_HOME variable in your environment to match the
location of your Java installation.


> Task :ui-android:installAndroidEmulatorSystemImage FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':ui-android:installAndroidEmulatorSystemImage'.
> Process 'command '/Users/dberry/Library/Android/sdk/tools/bin/sdkmanager'' finished with non-zero exit value 1

```

--- a/android-emulator-plugin/src/main/java/com/quittle/androidemulator/AndroidEmulatorPlugin.java
+++ b/android-emulator-plugin/src/main/java/com/quittle/androidemulator/AndroidEmulatorPlugin.java
@@ -187,7 +187,7 @@ public class AndroidEmulatorPlugin implements Plugin<Project> {
                                        final String name,
                                        final Action<AbstractExecTask<Exec>> configure) {
         return project.getTasks().create(name, Exec.class, (final Exec task) -> {
-            task.setEnvironment(emulatorConfiguration.getEnvironmentVariableMap());
+            task.environment(emulatorConfiguration.getEnvironmentVariableMap());
 
             configure.execute(task);
 

